test/cpp/jit/test_gpu.cpp

```diff
@@ -8975,8 +8975,8 @@ TEST_F(NVFuserTest, FusionMagicSchedulerRMSNormalization_CUDA) {
   const float kEps = 1e-6;
   Double* eps_ptr = IrBuilder::create<Double>(kEps);
 
-  std::vector<int64_t> input_shape{8, 56, NORM_SIZE};
-  std::vector<int64_t> norm_shape{NORM_SIZE};
+  std::vector<int64_t> input_shape{8, 56, (int64_t)NORM_SIZE};
```

Comment:
This is actually not the only place where we are blindly putting `size_t NORM_SIZE` into the init list of a `vector<int64_t>`. There's also `FusionMagicSchedulerRMSNormBackward_CUDA`

FYI: https://github.com/pytorch/pytorch/pull/73627/commits/c345c14c966a935ddddad3275bc05b47cc4b8078

I'm cool with either way to patch that.